fix #19, tutorial update pycall (ex10) (#21)
pdidev/tutorial: README.md

### Ex10. Post-processing the data in python

\attention
You need to have a version %PDI with the \ref pycall_plugin "Pycall plugin"
to do this exercise.

In this exercise, you will once again modify the YAML file only and use python
to post-process the data in situ before writing it to HDF5.
Here, you will write the square root of the raw data to HDF5 instead of the
data itself.

* Examine the YAML file and compile the code.

* Load the \ref pycall_plugin "Pycall plugin" and enable this plugin
  when the `loop` event is triggered.

Some variables of the python script inside `ex10.yml` are not defined.
The `with` directive of this plugin allows to specify input variables (parameters)
to pass to Python as a set of "$-expressions".
These parameters can be given as multiple blocks.

* Add a `with` block with the missing parameter to let the Python code process
  the data exposed in `main_field` for event `loop`.

* Use the keyword `exec` of \ref pycall_plugin "Pycall plugin" and decomment
  the python script.

Notice that the Decl'HDF5 configuration was simplified, no memory selection is

> applied, the `when` condition disappeared because it is done in the python script:

Reviewer: ```suggestion
applied. The `when` condition was moved to the python script:
```
